src/components/NotFound.less

```diff
@@ -0,0 +1,6 @@
+.error-placeholder__error-msg {
+    width: 100%;
+    height: 100%;
+    text-align: center;
+    margin-bottom: 40px;
```

Comment:
i think you could remove this `margin-bottom` and add `min-height: calc(100vh - 242px);` to put the footer close to the bottom of the page